frontends/web/src/api/account.ts

```diff
@@ -70,11 +74,11 @@ export const getAccounts = (): Promise<IAccount[]> => {
 };
 
 export type TAccountsBalanceByCoin = {
-    [key: string]: IAmount;
+    [key in CoinOrTokenCode]: IAmount;
```

Comment:
```suggestion
    [key in CoinOrTokenCode]?: IAmount;
```
otherwise it is typed that the backend alwas returns a map with all coins/tokens, but it only returns a map with used coins/tokens

<img width="488" alt="Screenshot 2024-05-30 at 09 46 28" src="https://github.com/BitBoxSwiss/bitbox-wallet-app/assets/546900/a38d4b3f-121d-4922-b186-8f1fa0b863ea">